tst/standard/SubfieldMatrixGroups.tst

```diff
@@ -30,50 +30,47 @@ gap> TestUnitarySubfieldSU := function(args)
 >   G := UnitarySubfieldSU(n, p, e, f);
 >   hasSize := HasSize(G);
 >   RECOG.TestGroup(G, false, Size(G));
->   return IsSubset(SU(n, p ^ e), GeneratorsOfGroup(G))
->          and DefaultFieldOfMatrixGroup(G) = GF(p ^ (2 * e))
->          and hasSize;
+>   Assert(0, IsSubset(SU(n, p ^ e), GeneratorsOfGroup(G)));
+>   #Assert(0, DefaultFieldOfMatrixGroup(G) = GF(p ^ (2 * e))); # FIXME
```

Comment:
@maxhauck can yo tell me what's going on here? Should it perhaps be like this?
```suggestion
>   Assert(0, DefaultFieldOfMatrixGroup(G) = GF(p ^ (2 * f)));
```
That at least works in the first two test cases. It fails the third one, though
```
gap> TestUnitarySubfieldSU([3, 5, 3, 1]);
Error, Assertion failure in
  DefaultFieldOfMatrixGroup( G ) = GF( p ^ (2 * f) ) at *stdin*:40 called from
<function "TestUnitarySubfieldSU">( <arguments> )
 called from read-eval loop at *stdin*:45
you may 'return;'
brk> DefaultFieldOfMatrixGroup( G );
GF(5^6)
brk> e;
3
brk> f;
1
```

Reply:
No, I am pretty certain it _should be_ `GF(p ^ (2 * e))` indeed - however, this is again a problem with the base field of the matrix group: The group can be written over `GF(p ^ (2 * f))` and so GAP returns `GF(p ^ (2 * f))` in the first two cases instead of the much larger `GF(p^ (2 * e))` (which it should actually be).

Reply:
OK, but then the tests perhaps should just be whether the `DefaultFieldOfMatrixGroup` is a subfield -- and that's already covered by the check in the line above. So I think it could just be removed.